code/game/objects/structures/tables_racks.dm

```diff
@@ -707,8 +707,8 @@
 	smoothing_groups = null
 	canSmoothWith = null
 	can_buckle = 1
-	buckle_lying = NO_BUCKLE_LYING
-	buckle_requires_restraints = TRUE
+	buckle_lying = 90
+	buckle_requires_restraints = FALSE
```

Comment:
I think you can safely remove this line.

Reply:
I can remove the buckle_requires_restraints line but not the buckle_lying line. Will do now.